install/kubernetes/helm/istio/values.yaml

```diff
@@ -403,7 +419,7 @@ tracing:
     tag: 1.5
   service:
     name: http
-    type: ClusterIP
+    type: LoadBalancer #change to NodePort, ClusterIP or LoadBalancer if need be
```

Comment:
This should be `ClusterIP` for the zipkin service.

Need to add a `uiType: LoadBalancer` for the UI (tracing) service.